docs/resources/api_service.md

```diff
@@ -9,6 +9,67 @@ description: |-
 
 An API Service is an abstraction which defines for locating and communicating with servers.
 
+<!-- schema generated by tfplugindocs -->
+## Schema
```

Comment:
This autogenerated schema section is now duplicating the information in the [argument reference below](https://github.com/hashicorp/terraform-provider-kubernetes/blob/706e53b1bd95d58c9a4aa1e3cca2fe7495843e3d/docs/resources/priority_class_v1.md#argument-reference) – so I think you'll need to remove all those sections.